manifests/mod/proxy.pp

```diff
@@ -30,7 +30,7 @@
   Optional[String] $apache_version     = undef,
   Optional[String] $package_name       = undef,
   String $proxy_via                    = 'On',
-  Optional[String] $proxy_timeout      = undef,
+  Optional[Integer[0]] $proxy_timeout      = undef,
```

Comment:
Nit: ideally this would be aligned witht he other parameters

Reply:
@robertc99 Think it would be best to do as @ekohl suggested and make this change backwards compatible so we don't have to immediately cut a major release.
Have created a ticket to track this and any other similar issues so that we can correct them when the next major release is made:
https://github.com/puppetlabs/puppetlabs-apache/issues/2291